pyproject.toml

```diff
@@ -47,7 +47,8 @@ dev-dependencies = [
 	"myst-nb>=1.0.0; python_version>='3.9'",
 	"nbmake>=1.5.4",
 	"numpy<1.25.0; python_version<'3.9'",
-	"numpy>=1.26.2,<2; python_version>='3.9'",
+	"numpy>=1.26.2,<2; python_version>='3.9' and python_version <'3.13'",
+	"numpy>=2; python_version>='3.13'",
```

Comment:
I don't think we need all of this anymore. I think we just need `"numpy<1.25.0; python_version<'3.9'",`, and then `"numpy>=1.26.2; python_version>='3.9'",`

Reply:
ooh yeah, good point! will update.

Reply:
As an aside, that does mean we'll need to update the version check logic in `flat_tensors_to_ndrrays` to check the numpy version rather than the python version.